src/MarkdownBuilder.php

```diff
@@ -76,7 +76,7 @@ public function h3($header)
      * @param string $text
      * @return $this
      */
-    public function blockqoute($text)
+    public function blockquote($text)
```

Comment:
can you add a bc method?

```
/**
 * @deprecated
 */
public function blockqoute($text) {
    return $this->blockquote($text);
}
```

so i can it release as a patch level update.

Reply:
Hello.
I was just passing by - hope this helps.
Your library is a usefull companion for data fixtures. Thanks.

Reply:
nice, thank you!